org.uqbar.project.wollok/src/org/uqbar/project/wollok/scoping/cache/WollokGlobalScopeCache.xtend

```diff
@@ -0,0 +1,11 @@
+package org.uqbar.project.wollok.scoping.cache
+
+import java.util.List
+import org.eclipse.emf.common.util.URI
+import org.eclipse.xtext.xbase.lib.Functions.Function0
+import org.uqbar.project.wollok.wollokDsl.Import
+import org.eclipse.xtext.resource.IEObjectDescription
+
+interface WollokGlobalScopeCache {
```

Comment:
Yes. I don't  know of a way besides having your own eclipse "Builder" or BuildParticipant to be notified about a file changed / being built.

If I remember correctly there is an xtext specific extension point for extending this. Something about BuildParticipants